atm.js

```diff
@@ -1,3 +1,133 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+  function getFieldInput(accountType){
+    var field = $("#" + accountType + ">input.input");
```

Comment:
i really like this, one function for getting input from either, just pass in the accoutType! 👍